### PR TITLE
Fix deprecation for v0.5 and adapt macro to expr changes

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,4 +1,5 @@
-julia 0.4
+julia 0.4+
```
Comment (tkelman) [inline REQUIRE]: why did you add the `+`? that makes the package not installable on 0.4

Comment (vchuravy) [inline REQUIRE]: I think in the first iteration I was planning on removing support vor `v0.4` completely before settling on `Compat`.
